lib/cylc/rundb.py

```diff
@@ -531,7 +531,7 @@ def select_checkpoint_id(self, callback, id_key=None):
 
         If id_key is specified, add where id == id_key to select.
         """
-        stmt = r"SELECT id,time,event FROM %s" % self.TABLE_CHECKPOINT_ID
+        stmt = r"SELECT id,time,event FROM checkpoint_id"
```

Comment:
Por qué?

Reply:
This keeps Codacy happy.

Reply:
Codacy causing more harm than good :disappointed:. If we are going to go down this route I think we need to do it everywhere rather than just a couple of places as a mixed approach is more likely to result in error down the line. I'd rather just delete Codacy comments for now, the constant approach is safer.

Reply:
I have changed the references to *one* table here - mainly because I am creating a new method using that table. I was going to to the rest in a separate PR, but happy to revert. Preference?

Reply:
I think using constants is safer and would rather not allow uninformed dribble from Codacy determine our development practices, but we can decide this in a future PR.

Reply:
I agree with @oliver-sanders - but it seems you've decided to resolve this in a future PR. (which is fine).